camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/CamusJob.java

```diff
@@ -448,10 +458,10 @@ public void sendTrackingCounts(JobContext job, FileSystem fs, Path newExecutionO
         }
         ObjectMapper mapper = new ObjectMapper();
         mapper.configure(DeserializationConfig.Feature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+
         ArrayList<EtlCounts> countsObjects =
             mapper.readValue(buffer.toString(), new TypeReference<ArrayList<EtlCounts>>() {
```

Comment:
Ideally I would like to pass `new TypeReference<ArrayList<EtlCountsForUnitTest>>()` in unit test. However, this doesn't seem to be easily doable. So I still pass `new TypeReference<ArrayList<EtlCounts>>()` here, and then in line 503, convert `EtlCounts finalEtlCounts` to `etlCountsClass`, which is `EtlCounts` normally, and `EtlCountsForUnitTest` in unit test.